xml/admin_ceph_upgrade.xml

```diff
@@ -616,12 +616,76 @@ ID   CLASS  WEIGHT   TYPE NAME       STATUS  REWEIGHT  PRI-AFF
    <title>Upgrade the &igw;</title>
    <para><remark>TBD</remark></para>
   </sect2>
-
+-->
   <sect2 xml:id="upgrade-ogw">
    <title>Upgrade the &ogw;</title>
-   <para><remark>TBD</remark></para>
+   <para>
+    In &productname; &productnumber;, the &ogw;s are always configured as
+    multi-site (see <xref linkend="ceph-rgw-fed"/> for more details). If you
```

Comment:
maybe instead of a multisite mention something like RGWs are always configured with a realm, which allows for multisite in the future.